### DISCUSSION TITLE
Concourse web isn't working when web replica is greater than 1

### DISCUSSION THREAD

**Discussion by oo00spy00oo** [concourse/concourse#6112] [Help & Support]
## Summary

Concourse web isn't working when web replica is greater than 1

## Expected results

When the replicas of concourse-web greater than 1, the Concourse CI is still working

## Actual results
Logging:

`concourse-web`

![image](https://user-images.githubusercontent.com/7278061/94989745-33b05600-05a1-11eb-904d-942f1eebe02b.png)

![image](https://user-images.githubusercontent.com/7278061/94989948-9f46f300-05a2-11eb-88bb-34f4b787bd81.png)

`concourse-worker`

![image](https://user-images.githubusercontent.com/7278061/94989879-0617dc80-05a2-11eb-976e-20b8e636b828.png)

## Triaging info

* Concourse version:
* Browser (if applicable):
* Did this used to work?

**Comment by andy-paine** [comment]
I don't see anything in those logs to suggest there is something wrong with Concourse itself (and particularly not the web node) - is there a particular reason you think it is linked to the number of replicas? It looks like your `ci` and `source-code` resources are not finding any versions when running the `check` commands.

**Comment by oo00spy00oo** [comment]
> I don't see anything in those logs to suggest there is something wrong with Concourse itself (and particularly not the web node) - is there a particular reason you think it is linked to the number of replicas? It looks like your `ci` and `source-code` resources are not finding any versions when running the `check` commands.

When I made web replicas return to 1, the CI worked again.

**Comment by chenbh** [comment]
There are several log entries in there for `"message": "atc.credential-manager.secret-not-found", ... "secret-name": "demo.gitlab"`. Assuming that `ci` and `source-code` are resources referring to private repos on GitLab, this would cause the resource to error / turn yellow and cause the build to be stuck in pending.

It also looks like there's multiple builds, were builds 1-15 run with a single web node? And how long was the pending build stuck for?

**Comment by jamieklassen** [comment]
converting to a discussion until we have a reproducible, per https://blog.concourse-ci.org/nobody-wants-a-stale-bot/#an-update-on-triage.
